src/otg/method/l2g/feature_factory.py

```diff
@@ -96,57 +96,93 @@ def _get_max_coloc_per_study_locus(
             .select("studyLocusId", "right_studyType", "geneId", "coloc_score")
         )
 
-        # Max LLR calculation per studyLocus AND type of QTL
+        # Max PP calculation per studyLocus AND type of QTL
         local_max = get_record_with_maximum_value(
             colocalising_study_locus,
             ["studyLocusId", "right_studyType", "geneId"],
             "coloc_score",
-        )
+        ).persist()
+
+        intercept = 0.0001
```

Comment:
The value of the maximum PP in the neighborhood is the log of the difference between the maximum PP for the gene and the maximum PP across genes. I add this intercept so that when these 2 match up I can calculate the log.

Reply:
I don't fully understand the logic here. The difference can be negative.